packages/babel-preset-react-server/package.json

```diff
@@ -29,5 +31,8 @@
     "babel-plugin-transform-es2015-template-literals": "^6.3.13",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-preset-react": "^6.5.0"
+  },
+  "devDependencies": {
+    "rimraf": "^2.2.8"
```

Comment:
Nice to avoid distributing `devDependencies` around, when possible.

Had just finally gotten rid of most packages `devDependencies`. ;)